pkg/kubelet/server/auth.go

```diff
@@ -99,6 +101,8 @@ func (n nodeAuthorizerAttributesGetter) GetRequestAttributes(u user.Info, r *htt
 			subresources = append(subresources, "healthz")
 		case isSubpath(requestPath, configz.DefaultConfigzPath):
 			subresources = append(subresources, "configz")
+		case isSubpath(requestPath, statusz.DefaultStatuszPath):
+			subresources = append(subresources, "statusz")
```

Comment:
This is a net-new path, so I don't think it needs to go under the fine-grained authz here. I think it can just go in the switch statement below (line 113).